src/current/cockroachcloud/network-authorization.md

```diff
@@ -1,45 +1,52 @@
 ---
-title: Network Authorization
+title: Network Authorization for CockroachDB Cloud clusters
```

Comment:
I can't tell, but is this page specific to dedicated?

Reply:
no, it has links for both.